devutils/update_submodule.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+UNGOOGLED_REPO=$(dirname $(dirname $(readlink -f ${BASH_SOURCE[0]})))/ungoogled-chromium
+
+CHROMIUM_VERSION=$(curl -sL https://raw.githubusercontent.com/Eloston/ungoogled-chromium/master/chromium_version.txt)
+UNGOOGLED_REVISION=$(curl -sL https://raw.githubusercontent.com/Eloston/ungoogled-chromium/master/revision.txt)
+
+if [ UNGOOGLED_REVISION != "0" ]
+then
+    UPDATED_TAG="${CHROMIUM_VERSION}-${UNGOOGLED_REVISION}"
+else
+    UPDATED_TAG="${CHROMIUM_VERSION}"
+fi
+    
+# Ensure the submodule is initialized.
+git submodule init
```

Comment:
So this script needs to be run from the root of this repo, right? Otherwise, I assume this will throw some git error?

Reply:
Yes, that script must be run from the root of this repo.

Reply:
Alright, as long as users don't accidentally run it elsewhere, this should be fine.